src/Turbo/README.md

```diff
@@ -176,6 +180,17 @@ $builder
         ]
     ]);
 ```
+> **NOTE Turbo ^7.0.0-beta.5:**
+> If Turbo ^7.0.0-beta.5 or lower (may be still valid with future version), there is a bug with turbo-frame and form submit. F.e., a link to add a product will work the 1st time, the frame will have the new product form, on submit, the frame will be updated using the products listing. At this point the new link won't work anymore. This is cache issue in Turbo. To avoid it add a ramdom value as get parameter, microtime(true) will make it unique for one client:
```

Comment:
We should verify, but I think this is fixed on beta 7.

Reply:
@weaverryan yes, I will test master or latest release later today. For the time, may I suggest to keep the note? Took some time to dig the issues reports and the turbo code :) I wish noone to have to do it ;)

Reply:
I confirmed, beta.7 works as expected.